src/main/java/org/opensearch/jobscheduler/rest/action/RestGetJobDetailsAction.java

```diff
@@ -61,39 +63,30 @@ public String getName() {
     public List<Route> routes() {
         return ImmutableList.of(
             // New Job Details Entry Request
-            new Route(PUT, String.format(Locale.ROOT, "%s/%s", JobSchedulerPlugin.JS_BASE_URI, "_get/_job_details")),
+            new Route(PUT, String.format(Locale.ROOT, "%s/%s", JobSchedulerPlugin.JS_BASE_URI, "_job_details")),
```

Comment:
Answer 1) /_plugins/_job_scheduler/_get/_job_details now changed to /_plugins/_job_scheduler/_job_details
 It has extra _get in its path. So if the API is already a GET Api then it does not make sense to keep _get path

Answer 2) Yes it was a mistake. 

The API is PUT because it updates job details of extension in job scheduler meta deta index. However, essentially what is does is get job details from extension from extension and then update in its index. That's the reason it is PUT on JS end